provider/marathon/marathon_test.go

```diff
@@ -257,6 +257,124 @@ func TestMarathonLoadConfigNonAPIErrors(t *testing.T) {
 				},
 			},
 		},
+		{
+			desc: "multiple ports",
```

Comment:
The test cases added to this test all seem to belong to `TestMarathonGetPort`.

Also, please replace the literal type definitions by the builder functions we're using by now. See other tests for how to do that.